src/Uno.UI.FluentTheme.v1/Uno.UI.FluentTheme.v1.Skia.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras" ToolsVersion="15.0">
+	<PropertyGroup>
+		<TargetFrameworks>netstandard2.0</TargetFrameworks>
+	</PropertyGroup>
+
+	<Import Project="../netcore-build.props" />
+	<Import Project="../targetframework-override.props" />
+
+	<PropertyGroup>
+		<NoWarn>$(NoWarn);NU1701</NoWarn>
+		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+		<AssemblyName>Uno.UI.FluentTheme.v1</AssemblyName>
+		<RootNamespace>Uno.UI.FluentTheme.v1</RootNamespace>
+		
+		<ProduceReferenceAssembly>false</ProduceReferenceAssembly>
+		<Deterministic>true</Deterministic>
+		
+		<UnoRuntimeIdentifier>Skia</UnoRuntimeIdentifier>
+	
+		<UseCommonOverridePackage>true</UseCommonOverridePackage>
+		<CommonOverridePackageId>Uno.UI</CommonOverridePackageId>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<ProjectReference Include="..\Uno.UI\Uno.UI.Skia.csproj" />
+	</ItemGroup>
+	
+	<ItemGroup>
+		<PackageReference Include="Uno.SourceGenerationTasks" />
```

Comment:
Consider marking it as PrivateAssets = "All". I don't think this reference is needed in runtime but only in buildtime.

Reply:
Uno packages these references without the nuget "pack" target which uses the PrivateAssets attribute. The modification is not needed.